source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -18,3 +18,31 @@ The global vulnerability database is created automatically, currently pulling da
 This database can be configured to be updated periodically, ensuring that the solution will check for the very latest CVEs.
 
 Once the global vulnerability database (with the CVEs) is created, the detection process looks for vulnerable packages in the inventory databases (unique per agent). Alerts are generated when a CVE (Common Vulnerabilities and Exposures) affects a package that is known to be installed in one of the monitored servers. A package is labeled as vulnerable when its version is contained within the affected range of a CVE.
+
+Scan types
+==========
+
+The results are presented as alerts and also stored in a database. You can check the last scan alerts, or query each agent's individual CVEs DB.
+
+The Vulnerability Detector module can run a scan on startup (**run_on_start** option) and with a fixed frequency (**interval**).
+In any of these cases, the packages that have been already scanned will wait until the **ignore_time** expires to be re-scanned.
+This results in a better performance and we avoid a repeated alert during a configurable time. We have then two different type of scans
+
+- Partial scans: only new packages are scanned, as long as **ignore_time** is still valid
+- Full scan: the first scan of an agent always is complete. But then, only when the **ignore_time** passes the module scans all available packages again
```

Comment:
I would reword this explanation in this way. `The first time, Vulnerability Detector scans every single package installed. But then, only when the configured **ignore_time** expires, the module will scan all the available packages again.`

Reply:
I've made the reword you suggest, but the passive voice seem to me more adequate. And the "module" word becomes unnecessary in that case.

Please, take a look at it and tell me.